staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles_test.go

```diff
@@ -408,6 +408,12 @@ func TestRestrictedProfile(t *testing.T) {
 						{
 							Name:  "dbg",
 							Image: "dbgimage",
+							SecurityContext: &corev1.SecurityContext{
```

Comment:
I think, we really need unit tests to cover what will be the expected behavior when container already contains some `Capabilities`, `SecurityContexts`, etc. As far as I can see, all our unit tests are using containers have no capability or security context.

But this can be done in another PR, I have already filed this https://github.com/kubernetes/kubectl/issues/1372.

Reply:
Actually, we do want to clear those based on my understanding of the KEP. I think that's what the "Empty SecurityContext" is referring to under the "Debug Container Behavior" column. So the idea is that we first empties the security context, and then set run-as-non-root and drop-all-capabilities.

![image](https://user-images.githubusercontent.com/18542826/219314121-e1f8508a-d161-403e-b17b-72500cb23da5.png)